### DISCUSSION TITLE
MutuallyExclusiveComponent?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1197] [Ideas]
Hi:

People are asking 
is it possible to have a mutually exclusive component during the collide of two components?
Currently we could have physic component to have this effect
but what in that case, we have to make this entity physically droppable
so it is possible to have a non physical component that can collide with other specific components like mutually exclusive not intervene 

Thanks

**Comment by AlmasB** [comment]
Hi, could you provide a step-by-step example for this please. Generally, for collision purposes it's the `BoundingBoxComponent` that is used to check, but an example would clarify what functionality is needed for your use case.
